src/scenes/ErrorScene.tsx

```diff
@@ -39,27 +41,22 @@ export function ErrorScene(): JSX.Element {
   const { tryAgain } = useActions(verificationLogic)
 
   const actionButtons = endUserError?.noRetry ? (
-    <Button onClick={terminate} block>
+    <Button fullWidth onClick={terminate}>
       Close
     </Button>
   ) : (
-    <>
-      <SButtonContainer>
-        <Button onClick={tryAgain} block>
-          Try again
-        </Button>
-      </SButtonContainer>
-      <SCancelContainer>
-        <LinkButton onClick={terminate}>Cancel</LinkButton>
-      </SCancelContainer>
-    </>
+    <Button color="primary" fullWidth onClick={tryAgain}>
+      Try again
+    </Button>
   )
 
   return (
     <SWrapper>
-      <StatefulIcon />
-      <H2>{endUserError?.title || 'Something went wrong!'}</H2>
-      <SCaption>{endUserError?.caption || 'Sorry, there was a problem with your request, please try again.'}</SCaption>
+      <SIconWrapper>
+        <StatefulIcon />
```

Comment:
@michaelfurutin this indicator bg is blue on mockups 
![image](https://user-images.githubusercontent.com/89008845/175542770-3f816cac-4583-4771-a12c-92e22721bd9f.png)

Reply:
added